README.md

```diff
@@ -4,11 +4,13 @@
 
 There are various configurations for testing to cover different versions of Keycloak and Kong.
 
-For `Kong v.2.8.5` and `Keycloak v.15.1.1`, run the following:
+For `Kong v.3.9.0` and `Keycloak v.15.1.1`, run the following:
```

Comment:
The following command is plainly for Kong 2.8.5.

Could include in the readme how to run w/ 3.9.0 and quarkus, or leave it to the user to cobble together (based on what is in the `test.yaml` GHA)